SPECS/kernel-uvm/config

```diff
@@ -2578,8 +2579,19 @@ CONFIG_NETWORK_FILESYSTEMS=y
 # CONFIG_NFS_FS is not set
 # CONFIG_NFSD is not set
 # CONFIG_CEPH_FS is not set
-# CONFIG_CIFS is not set
+CONFIG_CIFS=y
+CONFIG_CIFS_STATS2=y
+CONFIG_CIFS_ALLOW_INSECURE_LEGACY=y
```

Comment:
That looks scary...

Reply:
Yes, we need to check all dependent configs which have been enabled for CONFIG_CIFS.

Reply:
Kindly disable this config (set it to `# CONFIG_CIFS_ALLOW_INSECURE_LEGACY is not set`) since we do not want to enable support for older and less secure dialects.